scripts/testing/debug_go_emotions_labels.py

```
@@ -27,16 +27,16 @@ def install_dependencies():
 def debug_go_emotions():
     """Debug the actual GoEmotions dataset structure."""
     print("🔍 Debugging GoEmotions dataset structure...")
-    
+
     # Load the dataset
     go_emotions = load_dataset("go_emotions", "simplified")
-    
+
     print(f"\n📊 Dataset structure:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)